src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -103,31 +102,25 @@ private static AuthorConfiguration findMatchingAuthorConfiguration(
     /**
      * Associates {@code emails} to {@code author}, if provided and not empty.
      */
-    private static void setEmails(AuthorConfiguration config, Author author, List<String> emails) {
+    private static void setEmails(Author author, List<String> emails) {
         author.setEmails(new ArrayList<>(emails));
-        config.addAuthorEmailsAndAliasesMapEntry(author, author.getEmails());
```

Comment:
Sure. Here are the clarifications:

1. Yes, the emails, display name, aliases and gitId of the author will be added to the config, when we call `config.addAuthor`. This is done by the `setAuthorDetails` method within AuthorConfiguration. `setAuthorDetails` is invoked when `config.addAuthor` is called.

2. Yes, `config.addAuthorEmailsAndAliasesMapEntry` is still used. For example, it's used by within the above mentioned `setAuthorDetails` method.

3. Yes, we do not really need the methods `setEmails` and `setDisplayName`, since they are one liners. I will remove them.

Reply:
@fzdy1914 what do you think ? Thanks for your time on this PR

Reply:
LGTM